tollway/callbacks.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+import typer
+
+from tollway.constants import FILE_SUFFIX
+
+
+def total_event_callback(total_events: int):
```

Comment:
interesting how typer can work with callbacks.

Reply:
it is. Didn't know they existed until I started looking at documentation for something else.